docs/build/html/manual/index.html

```diff
@@ -167,14 +164,7 @@
   <div class="section" id="pytket-user-manual">
 <h1>Pytket User Manual<a class="headerlink" href="#pytket-user-manual" title="Permalink to this headline">¶</a></h1>
 <p>This manual refers to the following versions of pytket and extensions:</p>
-<div class="highlight-text notranslate"><div class="highlight"><pre><span></span>pytket == 0.11.0
-pytket-cirq == 0.12.0
-pytket-honeywell == 0.11.0
-pytket-projectq == 0.10.0
-pytket-pyquil == 0.12.0
-pytket-qiskit == 0.13.0
-pytket-qsharp == 0.13.0
-pytket-qulacs == 0.8.0
+<div class="highlight-text notranslate"><div class="highlight"><pre><span></span>pytket == 0.11.0n    # pytket-cirq == 0.12.0n    pytket-honeywell == 0.11.0n    pytket-projectq == 0.10.0n    pytket-pyquil == 0.12.0n    pytket-qiskit == 0.13.0n    pytket-qsharp == 0.13.0n    pytket-qulacs == 0.8.0n
```

Comment:
Did you generate this by running `./build-manual`? I don't understand why these versions are showing with extra letters on the end, and differently formatted. (When I did the same on this branch I got the expected formatting.)

Reply:
I did use build-manual. Could be a sphinx versioning issue, I'll run it again.